src/app/ReviewService/ReviewPrompter.Windows.cs

```diff
@@ -1,27 +1,29 @@
 ﻿#if WINDOWS
-using System;
 using System.Threading.Tasks;
 using Microsoft.Extensions.Logging;
+using Microsoft.Extensions.Logging.Abstractions;
 using ReviewService.Abstractions;
 
 namespace ReviewService;
 
 /// <summary>
 /// Windows implementation of <see cref="IReviewPrompter"/>.
 /// </summary>
-public sealed class ReviewPrompter : IReviewPrompter
+public class ReviewPrompter : IReviewPrompter
 {
 	private readonly ILogger _logger;
 
-	public ReviewPrompter(ILogger logger)
+	public ReviewPrompter(ILogger<ReviewPrompter> logger)
 	{
-		_logger = logger;
+		_logger = logger ?? NullLogger<ReviewPrompter>.Instance;
 	}
 
 	/// <inheritdoc/>
 	public Task TryPrompt()
 	{
-		throw new NotImplementedException();
+		_logger.LogDebug("Prompting is not implemented on windows.");
```

Comment:
Thinking about it, may be `Prompting for a review is not implemented on Windows.` could be better.

Reply:
Indeed